src/node/gov/handlers/proposals.h

```diff
@@ -758,7 +758,7 @@ namespace ccf::gov::endpoints
         "/members/proposals/{proposalId}:withdraw",
         HTTP_POST,
         api_version_adapter(withdraw_proposal),
-        detail::active_member_sig_only_policies("withdraw"))
+        detail::active_member_sig_only_policies("withdrawal"))
```

Comment:
In #5660, I'd made the new endpoint expect `ccf.gov.msg.type` to be `"withdraw"`, a totally new value. Since that's unreleased and an unnecessary change (and undocumented in the upstream API definition, since this is all within COSE), I think it makes sense to restore/reuse the existing `msg.type`.

Reply:
+1 for not breaking the `ccf_cose_sign1` script unnecessarily.